### PR TITLE
Bump `http` package version ^1.0.0

### PR DESCRIPTION
**Pull request by desistefanova**
Fixes: #1326

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 * Added support binary data type. ([#1320](https://github.com/realm/realm-dart/pull/1320))
 * Extended `ClientResetError` to return the `backupFilePath` where the backup copy of the realm will be placed once the client reset process has completed. ([#1291](https://github.com/realm/realm-dart/pull/1291))
 * Added `CompensatingWriteError` containing detailed error information about the writes that have been reverted by the server due to permissions or subscription view restrictions. The `Configuration.flexibleSync.syncErrorHandler` will be invoked with this error type when this error occurs ([#1291](https://github.com/realm/realm-dart/pull/1291)).
+* Bumped `http` package version to ^1.0.0 ([#1329](https://github.com/realm/realm-dart/pull/1329)).
```
Comment (nielsenko) [inline CHANGELOG.md:7]: Do we document package updates like this? If we do, then it should go in the internal section, no?

Comment (nielsenko) [reply CHANGELOG.md:7]: Otherwise this would qualify as minor version bump, which it definitely shouldn't

Comment (blagoev) [reply CHANGELOG.md:7]: no I don't think we need to do this

**Review by blagoev — APPROVED** [review]
It is ok,  after we remove the changelog entry

**Pull request merged**